extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/api/ui/UiResourceImpl.java

```diff
@@ -41,6 +42,8 @@
 import de.sovity.edc.ext.wrapper.api.ui.pages.policy.PolicyDefinitionApiService;
 import de.sovity.edc.ext.wrapper.api.ui.pages.transferhistory.TransferHistoryPageApiService;
 import de.sovity.edc.ext.wrapper.api.ui.pages.transferhistory.TransferHistoryPageAssetFetcherService;
+import de.sovity.edc.utils.catalog.DspCatalogServiceException;
```

Comment:
Why are the imports changing but not any other code?

Reply:
I forgot that one and the checkstyle didn't complain.
Deleted.